pack/tdc/tdcc.json

```
@@ -13,6 +13,313 @@
+        "back_name": "Curiosity Killed the Cat",
+        "back_text": "Each surviving investigator is defeated and suffers 1 physical trauma.",
+        "code": "11503",
+        "doom": 7,
+        "double_sided": true,
+        "encounter_code": "one_last_job",
+        "encounter_position": 3,
+        "faction_code": "mythos",
+        "flavor": "Your every step is dogged by both the Sheldons and the O'Bannions. It seems like the two warring gangs have put aside their beef to end your investigation.",
+        "illustrator": "Tiziano Baracchi",
+        "name": "Long Night",
+        "pack_code": "tdcc",
+        "position": 3,
+        "quantity": 1,
+        "text": "[reaction] After the last clue is discovered from an [[Arkham]] location: Discard the top 3 cards of the encounter deck. One at a time, spawn each [[Criminal]] enemy discarded this way at the nearest empty location.",
+        "type_code": "agenda"
+    },
+    {
+        "back_link": "11504b",
+        "clues": 2,
+        "code": "11504",
+        "encounter_code": "one_last_job",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "flavor": "You need to find out what happened to Tillinghast's shipment. With the increased gang activity all over Rivertown, you think that checking in with the Sheldons and the O'Bannions is the best place to start.",
+        "illustrator": "Borja Pindado",
+        "name": "Questioning the Gangs",
+        "pack_code": "tdcc",
+        "position": 4,
+        "quantity": 1,
+        "text": "*This act gets +1 [per_investigator] clue threshold.\n[fast] Spend X clues: <b>Parley.</b> Discard an engaged [[Criminal]] enemy with X remaining health at your location. Place 1 clue <i>(from the token pool)</i> on this act.\n<b>Objective</b> - At the end of the round, if the number of clues on this act equals or exceeds its clue threshold, advance.",
+        "type_code": "act"
+    },
+    {
+        "clues": 2,
+        "code": "11504b",
+        "encounter_code": "one_last_job",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "hidden": true,
+        "illustrator": "Mark Molnar",
+        "name": "Hibb's Roadhouse",
+        "pack_code": "tdcc",
+        "position": 4,
+        "quantity": 1,
+        "shroud": 3,
+        "text": "Easttown is connected to Hibb's Roadhouse.\n<b>Revelation</b> - Put Hibb's Roadhouse into play. Spawn the set-aside Sadie Sheldon enemy at Hibb's Roadhouse. Add the set-aside act 2a - \"The Sheldon Gang\" and act 3a - \"Face the Music\" to the act deck, then advance to act 2a.",
+        "traits": "Arkham. Front.",
+        "type_code": "location",
+        "victory": 1
+    },
+    {
+        "back_link": "11505b",
+        "clues": 2,
```

```suggestion
        "clues": 2,
        "clues_fixed": true,
```

I think since it's 2+, this should have a fixed value. Renders as 2 [per_investigator] right now.

---

Good thought, thanks.